render/canvas/src/lib.rs

```diff
@@ -170,30 +170,6 @@ impl BitmapData {
             context,
         })
     }
-
-    fn get_pixels(&self) -> Option<Bitmap> {
```

Comment:
Without this method, the whole struct becomes pretty useless. I think it can be removed.

Reply:
We still use `BitmapData.canvas` in `create_bitmap_pattern`. I think additional refactoring of the `canvas` backend should be left to a separate PR.